documentation/asciidoc/topics/ref_server_configuration_values.adoc

```diff
@@ -0,0 +1,113 @@
+[id='server-configuration-values_{context}']
+= {brandname} Server configuration values
+
+[role="_abstract"]
+{brandname} Server configuration values let you do the following:
+
+* Create caches at startup with any {brandname} configuration.
+* Add cache templates and use them to create caches on demand.
+* Specify one of the default JGroups stacks.
+* Pass custom JGroups stacks to {brandname} clusters.
+* Define {brandname} Server security realms.
+* Configure {brandname} Server endpoints with custom Hot Rod and REST connectors.
+* Add {brandname} Server data sources and credential keystores.
+
+[discrete]
+== {brandname} cache configuration
+
+.{brandname} cache
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/cache_manager_caches.yaml[]
+----
+
+.Cache template
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/cache_manager_cache_templates.yaml[]
+----
+
+[discrete]
+== Cluster transport
+
+.Default JGroups stack
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/jgroups_default.yaml[]
+----
+
+.Custom JGroups stack
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/jgroups_relay.yaml[]
+----
+
+[discrete]
+== {brandname} Server security realms
+
+[IMPORTANT]
+====
+Do not modify or remove the "metrics" security realm to avoid loss of service with your {brandname} cluster.
+====
+
+[NOTE]
+====
+The {brandname} chat currently supports property realms only.
```

Comment:
Typo: `chat/chart`

Reply:
Also, this isn't true ... any type of realm can  be configured by the user as they have full access to the server configuration.